web/jsonapi/data.go

```diff
@@ -94,13 +94,6 @@ func MarshalObject(o Object) (json.RawMessage, error) {
 	links := o.Links()
 	rels := o.Relationships()
 
-	o.SetID("")
-	o.SetRev("")
-	defer func() {
-		o.SetID(id)
-		o.SetRev(rev)
-	}()
-
```

Comment:
@nono

I removed this hack because it provoked some race-conditions that would break installation APIs that rely on the fact that the manifest should not be mutated by consumer of the API. Their may be other race-conditions of the same nature in the code.

The side effect is that the `_id` and `_rev` in the `data` field.

Reply:
I don't like that `_id` and `_rev` are present in the `data.attributes` field.